client/js/App.vue

```diff
@@ -34,4 +34,10 @@ export default {
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
 }
+
+@media (max-width: 600px) {
+  div > .v-application--wrap {
+    min-height: 100%;
+  }
+}
```

Comment:
Need a comment above this with the vuetify documentation on using vh and maybe a link to where it doesn't work on mobile so we have some context.